[Bug]: YESTERDAY() function in Formula transform needs to be removed from the docs

### Apache Hop version?

SNAPSHOT-20230223

### Java version?

openjdk version "11.0.18" 2023-01-17

### Operating system

Linux

### What happened?

`Caused by: org.apache.poi.ss.formula.eval.NotImplementedFunctionException: YESTERDAY`

the YESTERDAY() function is not implemented, needs to be removed from the docs, so this is a documentation issues, not a transform issue. 

### Issue Priority

Priority: 2

### Issue Component

Component: Documentation

--- a/docs/hop-user-manual/modules/ROOT/pages/pipeline/transforms/formula.adoc
+++ b/docs/hop-user-manual/modules/ROOT/pages/pipeline/transforms/formula.adoc
@@ -728,23 +728,6 @@ Applications shall support extracting the year from a date beginning in 1900. Th
 |YEARFRAC("1/1/2012","7/30/2012",3) |0.57808219|Fraction between same dates, using the Actual/365 basis argument. Uses a 365 day basis.
 |===
 
-==== YESTERDAY
-
-|===
-|*description*|Return the serial number of the previous day
-|*syntax*|YESTERDAY()
-|*returns*|Date
-|*semantics*|This returns the previous day's serial number, using current locale. This only returns the date, not the datetime value.
-|*constraints*|None
-|===
-
-*Examples*
-|===
-|expression|result|comment
-|TODAY()>YESTERDAY() |False|Every date YESTERDAY() changes, but we know it's less than today.
-|INT(YESTERDAY)=YESTERDAY() |True|YESTERDAY() returns an integer. WARNING: this test is allowed to fail if the locale transitions through midnight while computing this test; because YESTERDAY() is referenced twice, in some implementations this would result in a race condition) This is incredibly unlikely to occur in practice.
-|===
-
 === Financial
 
 ==== PMT

--- a/plugins/transforms/formula/src/main/resources/org/apache/hop/pipeline/transforms/formula/function/functions.xml
+++ b/plugins/transforms/formula/src/main/resources/org/apache/hop/pipeline/transforms/formula/function/functions.xml
@@ -3182,23 +3182,6 @@ System 2    The week containing the first Thursday of the year is the first week
 	</examples>
   </function>
 
-
-
-
-  <function>
-    <category>%Category.DateTime</category>
-    <name>YESTERDAY</name>
-    <description>Return the serial number of the previous day</description>
-    <syntax>YESTERDAY()</syntax>
-    <returns>Date</returns>
-    <constraints>None</constraints>
-    <semantics>This returns the previous day's serial number, using current locale. This only returns the date, not the datetime value.</semantics>
-    <examples>
-      <example><expression>TODAY()>YESTERDAY()</expression> <result>False</result> <level>1</level> <comment>Every date YESTERDAY() changes, but we know it's less than today.</comment></example>
-      <example><expression>INT(YESTERDAY)=YESTERDAY()</expression> <result>True</result> <level>1</level> <comment>YESTERDAY() returns an integer. WARNING: this test is allowed to fail if the locale transitions through midnight while computing this test; because YESTERDAY() is referenced twice, in some implementations this would result in a race condition) This is incredibly unlikely to occur in practice.</comment></example>
-    </examples>
-  </function>
-
   <function>
   	<category>%Category.DateTime</category>
   	<name>WEEKDAY</name>

